src/_provisional.scss

```diff
@@ -329,6 +333,133 @@
 	}
 }
 
+:not(form) .provisional {
+	&.gc-checklist {
+		ul {
+			padding-left: 0;
+		}
+
+		output {
+			font-size: 20px;
+			font-weight: bold;
```

Comment:
I suggest to style the `output` like we do for `panel` or `well`.

And the use of the "bold" should be assigned based on the content (HTML) that is going to be inserted inside the output element.